src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/iOSSpecific/PopUp.shared.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using XFPC = Xamarin.Forms.PlatformConfiguration;
+using XCTElement = Xamarin.CommunityToolkit.UI.Views.BasePopup;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.iOSSpecific
+{
+	public static class PopUp
+	{
+		public static readonly BindableProperty ArrowDirectionProperty = BindableProperty.Create(
+			"ArrowDirection", typeof(PopoverArrowDirection), typeof(BasePopup), PopoverArrowDirection.None);
+
+		public static void SetArrowDirection(BindableObject element, PopoverArrowDirection color) =>
+			element.SetValue(ArrowDirectionProperty, color);
+
+		public static PopoverArrowDirection GetArrowDirection(BindableObject element) =>
+			(PopoverArrowDirection)element.GetValue(ArrowDirectionProperty);
+
+		public static IPlatformElementConfiguration<XFPC.iOS, XCTElement> UseArrowDirection(this IPlatformElementConfiguration<XFPC.iOS, XCTElement> config, PopoverArrowDirection value)
+		{
+			SetArrowDirection(config.Element, value);
+			return config;
+		}
+	}
+
+	public enum PopoverArrowDirection
+	{
+		None,
+		Up,
+		Down,
+		Left,
+		Right,
+		Any,
+		Unknown
```

Comment:
What's the use of `Unknown`? @ahoefling?

Reply:
This was added in #854 by @geskill and I approved the changes.

I am not 100% sure what `Unknown` or `Any` is used for. Looking at the code this maps 1:1 with the iOS implementation. Since this is iOS specific I think it is okay to leave both of them in. If we get to a point where we want to make this a shared API instead of attached property we would need to determine if it is still applicable.

Reply:
Reference: https://developer.apple.com/documentation/uikit/uipopoverarrowdirection/